### DISCUSSION TITLE
Stableと同等かそれ以上の機能をもつスキンシステムが必要です！（We need a skin system that is equal to or surpasses the functionality of Stable!）

### DISCUSSION THREAD

**Discussion by H17-Kyle612** [ppy/osu#30262] [Ideas]
僕と僕の友だちは、Stableで出来ていたメニューやセレクト画面、カーソルなどのスキンが適用されないがためにLazerに移行できていません。
スキンシステムさえ完璧になればLazerに移行しない理由がありませんので、ぜひ早急に対応していただきたいです。

お忙しいのは分かってます。でもこれだけが移行の妨げになっています。

Translated text at DeepL->

My friends and I haven’t been able to fully transition to Lazer because menus, selection screens, cursors, and other skin elements that worked in Stable are not applied in Lazer. Once the skin system is perfected, we will have no reason to stay with Stable, so we kindly ask for this to be addressed as soon as possible.

We understand you’re busy, but this is the only thing holding us back from switching to Lazer.

**Comment by peppy** [comment]
skinning outside of gameplay is not a priority as there's still many other things to do. but you'll see things like song select improve soon.

**Reply by H17-Kyle612** [reply]
「but you'll see things like song select improve soon.」
これが聞けて安心しました。

私にはStableをすでに超えた完成度に見えます笑
Modsの説明のように一部翻訳されていない箇所はありますが、ほんとそれだけです。
Issueをすべて片づけるつもりなら相当な時間がかかりそうですが、、、

LazerがスタンダートなOsu!になるのはいつ頃を予定しているんでしょうか？
開発引き続き頑張ってください、応援してます！


Translated text at DeepL->

I feel reassured after hearing that 'but you'll see things like song select improve soon.'

To me, it already seems more complete than Stable, haha. There are only a few untranslated parts, like the mod descriptions, but that's really it. If you're planning to fix every issue, it seems like it will take quite a bit of time...

When is Lazer expected to become the standard version of osu!? Keep up the great work on the development, I'm rooting for you!

**Reply by peppy** [reply]
The timeline is flexible and we hope each person can make that decision for themselves. It depends on which features you need and how much you enjoy lazer compared to stable 😉

We'll keep trying our best
